File: inzage-cms/backend/app/schemas/ond.py
from datetime import datetime

from pydantic import BaseModel, ConfigDict


class Ond(BaseModel):
    ond_cd: int
    titel: str
    omschrijving: str
    sort_key: int
    notitie: str | None
    ts_mut: datetime
    user_nm: str


class OndIn(BaseModel):
    omschrijving: str
    titel: str
    sort_key: int
    notitie: str | None = None


class OndMinimalList(BaseModel):
    ond_cd: int
    titel: str

    model_config = ConfigDict(from_attributes=True)

File: inzage-cms/frontend/src/types/Ond.ts
export type Ond = {
  ond_cd: number
  titel: string
  omschrijving: string
  sort_key: number
}

export type EvtpOnd = {
  evtp_ond_cd: number
  evtp_cd: number
  ond: number
}

export type OndTree = {
  evtp_ond_cd: number
  ond_cd: number
  titel: string
}
